# PixelCoordinates.msg
int32[] x_coords
int32[] y_coords
